lib/bot.js

```diff
@@ -143,6 +143,13 @@ class Bot {
         const prelude = `Command sent from Discord by ${username}:`;
         this.ircClient.say(ircChannel, prelude);
         this.ircClient.say(ircChannel, text);
+      } else if (message.attachments) {
```

Comment:
You won't reach the else statement though, so nothing will actually happen with the message.

I guess it's fine if Discord doesn't include an empty attachments array for normal messages, but might be good to have nonetheless. What do you think?